### PR TITLE
chore(module): add nuxt-viewport

### PR DESCRIPTION
**Pull request by mvrlin**
#120

### CODE REVIEW

**Comment by mvrlin** [comment]
#120

File: modules/nuxt-viewport.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: nuxt-viewport
+description: Define custom viewports for your Nuxt project
+repo: mvrlin/nuxt-viewport
+npm: nuxt-viewport
+icon: nuxt-viewport.svg
+github: https://github.com/mvrlin/nuxt-viewport
+website: https://github.com/mvrlin/nuxt-viewport
+learn_more: ''
+category: Libraries
```
Comment (pi0) [inline modules/nuxt-viewport.yml:9]: Library is for modules specifically integrate a js/vue library. We might use extensions or ui for this module

Comment (mvrlin) [reply modules/nuxt-viewport.yml]: @pi0 Done :)

**Comment by pi0** [comment]
Thanks!

**Pull request merged**